site/app/templates/forum/RenderPost.twig

```diff
@@ -1 +1,25 @@
-{% if render_markdown %}{% include "misc/Markdown.twig" with { content: post_content } only %}{% else %}{{ post_content }}{% endif %}
\ No newline at end of file
+{% if render_markdown %}
+    {% include "misc/Markdown.twig" with { content: post_content } only %}
+{% else %}
+    {{ post_content }}
+{% endif %}
+{% if post_attachment is defined and post_attachment.exist == true %}
+    <table>
+        <tbody class="history-attachment-table", id="history-table-{{ edit_id }}">
+            {% for row in post_attachment.params.unencoded_data %}
+            {% if row[2] is defined %}
+                <tr>
+                    <td style="padding: 20px">
+                        <img src="{{ row[0] }}", alt="Click to view attachment in popup", title="Click to view attachment in popup", class="attachment-img", onclick="openInWindow(this)"/>
```

Comment:
Same comment about line length here.  Additionally, HTML attributes should be separated by spaces, not commas.  Apply throughout.